### DISCUSSION TITLE
Use Start11 to restore the windows10 start menu

### DISCUSSION THREAD

**Discussion by gailium119** [valinet/ExplorerPatcher#278] [Ideas]
I discovered recently that Start11 can restore the win10 start menu.I think you can put it into explorerpatcher using the same way of using StartAllback64.dll.

**Comment by valinet** [comment]
I don’t understand what you mean by “the same as”. Disassembling programs is not like playing with LEGO, change something there and off it works. Furthermore, if you look closely, Start11 reimplemented the menu, it’s not the code from Microsoft’s DLL, they reimplemented it and make it look almost identical, same as I made sws almost exactly like the Windows 10 switcher. I don’t think it is the original menu because live tiles do not work. So, what exactly is it that you are suggesting? Furthermore, I don’t want to duplicate all the functionality from all places in here. I prefer to focus more on original development, lately. Thanks.

**Comment by gailium119** [comment]
Well,I mean you can add a option to use the Start11 menu instead of the original one: Hook explorer to open the start11 menu instead of startmenuexperiencehost.exe when clicking the start button

**Reply by valinet** [reply]
But why can’t Start11 do that?

**Reply by gailium119** [reply]
Because it isn't free and gpl-licensed like Explorerpatccher.

**Reply by valinet** [reply]
What?

**Reply by valinet** [reply]
What does the license have to do with Stardock adding some feature to their app…?

**Reply by gailium119** [reply]
> What does the license have to do with Stardock adding some feature to their app…?

it is not a free app ,i'm afraid, so it's impossible for me to use it in install.wim

**Comment by gailium119** [comment]
Managed it myself. Took me a week of my spare time to crack start11 .

**Reply by valinet** [reply]
So you can have the Windows 10 Start button open Start11?

**Reply by gailium119** [reply]
No,Start11 is only useful in the start menu,but it is compatible with ExplorerPatcher,so I use them together
